packages/ember-self-focused/addon/services/focus-manager.js

```diff
@@ -29,7 +28,9 @@ export default Service.extend({
      */
     this._removeTabIndex = this._removeTabIndex.bind(this);
 
-    run.scheduleOnce('afterRender', this, this.set, '_isFirstRender', false);
+    requestAnimationFrame(() => {
```

Comment:
Specifically, add this to the top of this file:

```js
import { registerWaiter } from '@ember/test';
import { DEBUG } from '@glimmer/env';

let activeScheduledTasks = 0;

if (DEBUG) {
  // wait until no active rafs
  registerWaiter(() => activeScheduledTasks === 0);
}
```

Then whenever you are going to rAF do:

```js
activeScheduledTasks++;
requestAnimationFrame(() => {
  activeScheduledTasks--;
  // the rest of the code
});
```

Reply:
This is the technique we employ in [ember-app-scheduler](https://github.com/ember-app-scheduler/ember-app-scheduler/blob/master/addon/scheduler.ts#L150).

Reply:
- [x] done